.github/workflows/rav1e.yml

```diff
@@ -368,12 +359,10 @@ jobs:
           curl -LO "$LINK/$SCCACHE_FILE.tar.gz"
           tar xzf "$SCCACHE_FILE.tar.gz"
           echo "$Env:GITHUB_WORKSPACE/$SCCACHE_FILE" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
-      - name: Install ${{ matrix.toolchain }}
+      - name: Install ${{ matrix.target }} target
         uses: dtolnay/rust-toolchain@stable
         with:
-          profile: minimal
-          toolchain: stable-${{ matrix.target }}
```

Comment:
It turns out that this is the pattern for setting a default target that behaves as expected.